MapleServer2/Servers/Game/FieldManager.cs

```diff
@@ -102,16 +102,15 @@ public FieldManager(int mapId)
                 actors.Add(RequestFieldObject(new InteractActor(actor.Uuid, actor.Name, actor.Type) { }));
             }
             AddInteractActor(actors);
-            
-            // TODO: Someone needs to update parser to include map name
-            /*string mapName = MapMetadataStorage.GetMetadata(mapId).Name;
-            Triggers = triggerLoader.GetTriggers(mapName).Select(initializer =>
+
+            string mapName = MapMetadataStorage.GetMetadata(mapId).Name;
+            Triggers = TriggerLoader.GetTriggers(mapName).Select(initializer =>
             {
                 TriggerContext context = new TriggerContext(this, Logger);
                 TriggerState startState = initializer.Invoke(context);
                 return new TriggerScript(context, startState);
-            }).ToArray();*/
-            Triggers = new TriggerScript[0];
+            }).ToArray();
+            Triggers = Array.Empty<TriggerScript>();
```

Comment:
This was a temporary replacement to the above comment. Now that you have uncommented it, you want to remove this.

Reply:
Makes sense 🤦‍♂️